mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/building/impl/PojoIdentityMappingCollectorImpl.java

```diff
@@ -48,11 +50,11 @@
 			PojoEntityTypeAdditionalMetadata entityTypeMetadata,
 			PojoMappingHelper mappingHelper,
 			IndexedEntityBindingContext bindingContext,
-			boolean implicitProvidedId) {
+			BeanHolder<? extends IdentifierBridge<Object>> identifierBridge) {
 		this.typeModel = typeModel;
 		this.mappingHelper = mappingHelper;
 		this.bindingContext = bindingContext;
-		this.implicitProvidedId = implicitProvidedId;
+		this.identifierBridge = identifierBridge;
```

Comment:
`providedIdentifierBridge` would be less ambiguous. For other mappers, the identifier bridge is set up differently.